ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -1607,6 +1607,17 @@ static const func_info_t func_infos[] = {
 	F0("imagesetinterpolation",					MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_TRUE),
 	F1("imageresolution",						MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_TRUE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_LONG | MAY_BE_ARRAY_OF_LONG),
 
+	/* ext/spl */
+	F1("class_implements",						MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_STRING),
+	F1("class_parents",							MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_STRING),
+	F1("class_uses",							MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_STRING | MAY_BE_ARRAY_OF_STRING),
+	F0("iterator_apply",						MAY_BE_FALSE | MAY_BE_LONG),
```

Comment:
```suggestion
	F0("iterator_apply",						MAY_BE_NULL | MAY_BE_LONG),
```

Reply:
No, the signature I wrote should be correct, according to your earlier comments

> I've cleaned up the code a bit in 5acedab. That means iterator_count() and iterator_to_array() **can't return null.**

```php
php > var_export(iterator_count());
Warning: iterator_count() expects exactly 1 parameter, 0 given in php shell code on line 1
false
```

Reply:
Oh, I changed the wrong function